docs/docs/user_guide/sidebar/common/studies/studies_page.md

```diff
@@ -6,29 +6,25 @@ sidebar_label: Studies Page
 
 The Studies page provides a central location for creating Studies, uploading files, managing Study permissions, and creating new Workspaces.
 
-### Changing Organization Study Permissions
-To change permissions for an Organization Study:
+### Changing organization's Study permissions
```

Comment:
"Organization" is a category of studies, similar to "My Studies" and "Open Data". The term "organization's" implies that the study belongs to some organization, which might not be 100% valid.

suggestion: Perhaps change this to say "Changing Organization-categorized Study permissions"..?

Reply:
Thanks for the feedback. Incorporated the change.